hack/run-test-install-e2e.sh

```diff
@@ -2,6 +2,8 @@
 
 source hack/common.sh
 
+ENABLE_CLEANUP="${ENABLE_CLEANUP:-false}"
+
```

Comment:
the install suite should always try the uninstall stage (there are legit tests here) BUT should never try to cleanup